saleor/order/actions.py

```diff
@@ -786,7 +788,9 @@ def create_fulfillments(
         else FulfillmentStatus.WAITING_FOR_APPROVAL
     )
     for warehouse_pk in fulfillment_lines_for_warehouses:
-        fulfillment = Fulfillment.objects.create(order=order, status=status)
+        fulfillment = Fulfillment.objects.create(
```

Comment:
Setting `tracking number` previously we were also trigger hook: `tracking_number_updated`. We don't have any webhook for. such event but we have a payment plugin (`NPAtobaraiGatewayPlugin`)that listen to this event. AFAIR this payment plugin was using tracking number to process a payment. I think that to kepp an consistency, we should trigger hook here.